docker/courtlistener/.env

```diff
@@ -0,0 +1,2 @@
+JUDGE_PICS_PATH=/usr/local/lib/python2.7/site-packages/judge_pics
+SEAL_ROOKERY_PATH=/usr/local/lib/python2.7/site-packages/seal_rookery
```

Comment:
Still no luck getting these installed in /opt?

Reply:
correct.

Reply:
What's going wrong?

Reply:
I can install into opt but I still need the same path to link to python2.7 in Django.

Reply:
I see. Thanks for trying.